hack/hypernetes/README.md

```diff
@@ -0,0 +1,46 @@
+# Deployment scripts for hypernetes
+
+## System requirements
+
+The script is only tested on CentOS 7.2 with at least 8GB memory. It will install latest hypernetes with hyperd and OpenStack Mitaka.
+
+Note: The script should be run as `root`.
+
+## Usage
+
+1. Setup environment variables:
+
+```sh
+export HOSTNAME="k8s"
+export IF_NAME="eth0"
+export IF_IP=""
+export GOPATH="/gopath"
+```
+
+2. Clone hypernetes to go path:
+
+```sh
+mkdir -p ${GOPATH}/src/k8s.io
+# Run 'yum -y install git' if git is not installed.
+git clone https://github.com/hyperhq/hypernetes.git ${GOPATH}/src/k8s.io/kubernetes
+```
+
+3. Run the script to install hypernetes
+
+```sh
+cd ${GOPATH}/src/k8s.io/kubernetes
+hack/local-up-hypernetes.sh
+```
+
+## QA
```

Comment:
It worth to add a note that this script may easy to be failed due to network issue, but user can fix it by re-run or execute scripts separately?